webapp/vue.config.js

```diff
@@ -1,9 +1,11 @@
+const CompressionPlugin = require("compression-webpack-plugin");
+
 module.exports = {
   productionSourceMap: false,
   devServer: {
     proxy: {
-      '/api': {
-        target: 'http://localhost:3000',
+      '/api/v1/ts': {
+        target: 'http://192.168.0.61',
```

Comment:
Mhh, not sure if this should be committed. Does localhost not work?

Reply:
If you want to test against a running backend on the esp, you need to enter the proper IP